recipes/insurveyor/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "insurveyor" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kensung-lab/INSurVeyor/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 33c85157892d3256abc96bb2a9053f05da9dcf55befeed720e965577db0b78b5
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - htslib ==1.13
+  run:
+    - htslib ==1.13
+    - pysam ==0.16.0.1
+    - pyfaidx ==0.5.9.1
+    - numpy
+
+test:
```

Comment:
can you please include a test similar to `surveyor.py --help`

Reply:
Should I simply add "surveyor.py --help" as a test?

Reply:
Added to run_test.sh